docs/docs/deploy/baremetal.md

```diff
@@ -242,6 +251,22 @@ yarn rw deploy baremetal --first-run
 
 If there are any issues the deploy should stop and you'll see the error message printed to the console. Assume it worked, hooray! You're deployed to BAREMETAL.
 
+### Starting on reboot
+The `pm2` service requires some system "hooks" to be installed so it can boot up using your systems service manager.  Otherwise, your services will need to be manually started again on reboot.  These steps only need to be run the first time you deploy to a machine.
+
+1. Ssh into your server as you did for the "Server Setup".  Navigate to your source folder.  For example `cd /var/www/example`
+2. Run the command `yarn pm2 startup`.  You will see some output similar to the output below.  Select this output starting with `sudo` and run it.  *Note* you will be using sudo so will need the root password to complete this
```

Comment:
```suggestion
2. Run the command `yarn pm2 startup`.  You will see some output similar to the output below. See the output after "copy/paste the following command:"? You'll need to do just that: copy the command starting with `sudo` and then paste and execute it. *Note* this command uses `sudo` so you'll need the root password to the machine in order for it to complete successfully.
```

I had to re-read that section of a couple of times to get what it was saying...I think this makes it a little clearer (I also included a little section below that explains it again with the actual command).

Reply:
Thank you.  It's good to have a second set of eyes on this for clarity!